LEAF_Request_Portal/sources/FormStack.php

```diff
@@ -110,12 +110,12 @@ public function importForm($overwiteExisting = false)
         }
         if (isset($_POST['formPacket']))
         {
-            $formPacket = json_decode($_POST['formPacket'], true);
+            $formPacket = XSSHelpers::scrubObjectOrArray(json_decode($_POST['formPacket'], true));
```

Comment:
Need to revert this one and the one below it as it's necessary for functionality. Might need further discussion.

Reply:
Would it be better to do `sanitizeHTML` or `sanitizeHTMLRich`? Or do we need to try a different approach?

Reply:
Totally different approach. The issue is that importForm() is used as part of the Form Library system, where admins would be able to include and deploy custom HTML/JS.